.taskcluster.yml

```diff
@@ -254,13 +285,25 @@ tasks:
               features:
                 taskclusterProxy: true
               maxRunTime: 3600
-              image: "mozilla/taskboot:0.2.5"
+              image: "mozilla/taskboot:0.2.7"
               env:
                 TASKCLUSTER_SECRET: project/relman/rust-code-analysis/deploy
               command:
                 - "/bin/sh"
                 - "-cx"
-                - "taskboot github-release mozilla/rust-code-analysis ${version} --asset ${linux_artifact}:public/${linux_artifact} ${windows_artifact}:public/${windows_artifact}"
+                - "git clone --recursive --quiet https://github.com/SoftengPoliTo/rust-code-analysis &&
+                   cd rust-code-analysis &&
+                   taskboot retrieve-artifact --output-path=. \
+                            --artifacts=public/book.tar.gz &&
+                   tar xfz book.tar.gz -C rust-code-analysis-book &&
+                   git config --global user.email you@example.com &&
```

Comment:
Nit: move the `git config` commands right after the `cd`, so it's clear it's related.